packages/image/src/Image.js

```diff
@@ -20,7 +20,8 @@ Image.propTypes = {
   }),
   width: PropTypes.string,
   height: PropTypes.string,
-  fit: PropTypes.string
+  fit: PropTypes.string,
+  css: PropTypes.object
```

Comment:
`styled-components` also works with `string` or with the ``css` `;`` helper (which returns an `array`).
Let's accept those as well.

```suggestion
  css: PropTypes.oneOfType([
    PropTypes.object,
    PropTypes.string,
    PropTypes.array
  ])
```